.tekton/pipeline-build.yaml

```diff
@@ -352,7 +352,7 @@ spec:
             value: clamav-scan
 
           - name: bundle
-            value: quay.io/konflux-ci/tekton-catalog/task-clamav-scan:0.1@sha256:59a538a9c8affbfc584ddbe57468d8ce59c4830b37a472bc98ab8f46df82afce
+            value: quay.io/konflux-ci/tekton-catalog/task-clamav-scan:0.1@sha256:0f7df2d188af2a18b47a927ec1502391d24a3617b04db10a6b229a983de67d08
```

Comment:
This is an older image that contains the bug that is fixed in [newer images](https://quay.io/repository/konflux-ci/tekton-catalog/task-clamav-scan?tab=tags&tag=latest). We want to use images newer `59a538a`, not older.